packages/primitives/list/src/List.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import type { TStyle } from 'stili'
+import { normalizeWebStyle } from 'stili'
+import { component, startWithType, mapWithProps } from 'refun'
+import type { TList } from './types'
+
+export const List = component(
+  startWithType<TList>(),
+  mapWithProps(() => {
+    const style: TStyle = {
+      _webOnly: { listStyleType: 'none' },
```

Comment:
in order not to have error in native, this styling is only relevant for web
also it follow the logic from [stili](https://github.com/bubble-dev/_/blob/master/packages/stili/src/types.ts)

Reply:
but there's a specific native file 🤔
isn't this meant for files that are universal?

Reply:
its just the way stili is build `_webOnly` is just a type that contains types that are only relevant for web, does it make sense?

Reply:
it's just not clear to me when whether we should use it only for universal files or always.
summoning our tooling master @irminakacz

Reply:
i think its an "always" kinda thing, because its only TS thing

Reply:
@moriaam I see that readme for `stili` is actually wrong, so probably it's where the confusion comes from.
Right now `stili` exports `TStyle` that is universal and two methods `normalizeWebStyle` and `normalizeNativeStyle` that are not universal.

In web file you use `normalizeWebStyle` and pass `TStyle` to it to transform it to web styles (it will for example ignore everything under `_nativeOnly` and add `px` to pixel values).

In native you use `normalizeNativeStyle` and pass the same `TStyle` to it, but you'll get different output.

But it's ok, because in primitives we will always have separate web & native implementation. The universal part is the `TStyle`.

Reply:
We could have the TStyle in separate file and just pass it both to web and native files. But I guess you're not using it in native, because `Block` doesn't have any margin/padding set, right @DonGideon?

Reply:
exactly

Reply:
@DonGideon I think that as long as we don't allow to pass any styles from outside to List primitive it's perfectly fine 👍